src-web/components/kappnav/JobView.jsx

```diff
@@ -302,10 +313,10 @@ class JobView extends React.Component {
   }
 } // end of JobView component
 
-JobView.propTypes = {
-  appNavConfigData: PropTypes.object.isRequired,
-  namespace: PropTypes.string.isRequired,
-  namespaces: PropTypes.array.isRequired
-}
-
-export default JobView
+export default connect(
+  (state) => ({
+      baseInfo: state.baseInfo,
+  }),
+  {
+  }
+)(JobView);
```

Comment:
We should be consistent on whether we end a line with `;` or not. The future enablement of ESLint will flag the inconsistency.